src/components/ui/check-chip-group.svelte

```diff
@@ -0,0 +1,61 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+  import itemAmount from 'ui/utils/item-amount.js';
+
+  export let classname = '';
+  export let chipwrapperclass = '';
+  export let chipclass = '';
+  export let inputclass = '';
+  export let small = false;
+  export let outline = false;
+
+  export let values;
+  export let labels = null;
+  export let name;
+  export let maxChecked = null;
+  $: currentChecked = values.reduce((acc, elt) => acc + elt.checked, 0);
+  const titleObj = { title: `Can only select ${itemAmount(maxChecked, 'value')}.` };
+
+  if (values.length === 0) {
+    console.error('Must have at least one value in the checkbox group.');
+  }
+
+  if (labels !== null && values.length !== labels.length) {
+    console.error('Must have as many labels as there is values.');
+  }
+
+  const dispatch = createEventDispatcher();
+  function checkLimit(evt) {
+    if (maxChecked !== null && currentChecked + evt.target.checked  > maxChecked) {
+      evt.preventDefault();
+      return false;
+    }
+    return true;
```

Comment:
no need to return true. just move the dispatch here because it looks ugly in the markup.
In event handlers, returning false prevents default behavior and the propagation of the event. If nothing is returned, it is treated as true.